perception/lidar_transfusion/schema/transfusion.schema.json

```diff
@@ -21,6 +21,12 @@
           "default": "fp16",
           "enum": ["fp16", "fp32"]
         },
+        "cloud_capacity": {
+          "type": "integer",
+          "description": "Capacity of the point cloud buffer (should be set to at least the maximum theoretical amount of points).",
```

Comment:
"number of points"

Reply:
Addressed in [fa1abeb](https://github.com/autowarefoundation/autoware.universe/pull/7936/commits/fa1abebd498d8d134c408c2c9fa92b890581ca96)